wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -49,9 +53,21 @@ type MonadWalletHistory ctx m =
     , MonadTxHistory m
     )
 
+newtype WalletHistory =
```

Comment:
could you split adding `WalletHistory` and `WalletHistorySize` in a separate commit?

Reply:
The less PRs I have to look after, the easier it is for me, I'd rather not split